tests/test_derive_source_chain.rs

```diff
@@ -0,0 +1,69 @@
+use std::sync::Arc;
+
+use miette::{miette, Diagnostic, Report};
```

Comment:
looks like tests are failing because of this `Report` being unused.

Reply:
Oops, fixed!